framework/mongo/validators.py

```diff
@@ -7,3 +7,31 @@
 def string_required(value):
     if value is None or value == '':
         raise ValidationValueError('Value must not be empty.')
+    return True
+
+
+def choice_in(choices, ignore_case=False):
+    """
+    Validate that the option provided is one of a restricted set of choices
+
+    Returns a callable that can be used as a validator in ModularODM
+    :param choices: An iterable of choices allowed for use
+    :param bool ignore_case: If True, perform case-insensitive comparison (for strings) and otherwise match as-is
+    :return:
+    """
+
+    if ignore_case is True:
```

Comment:
You can; I just had a bad experience once with someone who liked to pass in `"false"` instead of `False`. I've been over-specifying ever since. :)